lib/charms/mongodb/v0/mongodb_backups.py

```diff
@@ -177,9 +177,7 @@ def _resync_config_options(self, pbm_snap):
         ):
             with attempt:
                 pbm_status = self._get_pbm_status()
-                if isinstance(pbm_status, MaintenanceStatus) or isinstance(
-                    pbm_status, WaitingStatus
-                ):
+                if isinstance(pbm_status, (MaintenanceStatus, WaitingStatus)):
```

Comment:
Hi @andreyrcdias thanks for helping us improve our code :)

The code in `lib/charms/mongodb` are part of our [charm libraries](https://juju.is/docs/sdk/libraries). When changes are made in the libary it is important to bump the `LIBPATCH` constant at the top of the file.

Definition of LIBPATCH:
> Current patch version of the library. Must be updated each time a new version of the library is published to Charmhub.